src/store/modules/accounts/hdWallet.js

```diff
@@ -48,7 +49,12 @@ export default {
     create({ state, commit }, isRestored = false) {
       commit(
         'accounts/add',
-        { idx: state.nextAccountIdx, type: ACCOUNT_HD_WALLET, isRestored },
+        {
+          idx: state.nextAccountIdx,
+          type: ACCOUNT_HD_WALLET,
+          isRestored,
+          protocol: PROTOCOL_AETERNITY,
```

Comment:
Ideally, protocol is a function parameter as we introduce multiple buttons to create protocol-specific accounts.